<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attemptQuiz</class>
 <widget class="QDialog" name="attemptQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="indexLabel">
       <property name="styleSheet">
        <string notr="true">font: 20pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>1/10</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLCDNumber" name="clockDisplay">
       <property name="digitCount">
        <number>7</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="questionBodyLabel">
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Body</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="answerLabel">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Short answer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="openAnswerEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="chooseBetweenLabel">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Times New Roman&quot;;</string>
       </property>
       <property name="text">
        <string>Choose between:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="answer1">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>cf</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="answer2">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="answer3">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="answer4">
           <property name="styleSheet">
            <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
           </property>
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextQuestionButton">
       <property name="text">
        <string>Next question&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
